test/e2e/common.go

```diff
@@ -287,6 +287,24 @@ func CheckAffinityGroupsDeleted(client *cloudstack.CloudStackClient, affinityIds
 	return nil
 }
 
+func GetHostCount(client *cloudstack.CloudStackClient, zoneName string) int {
+	pz := client.Zone.NewListZonesParams()
+	pz.SetName(zoneName)
+	listZonesResponse, err := client.Zone.ListZones(pz)
+	Expect(err).To(BeNil(), "error listing zones")
+	Expect(listZonesResponse.Count).To(Equal(1), "multiple zones resolve to zone name %s", zoneName)
+	zoneId := listZonesResponse.Zones[0].Id
+
+	ph := client.Host.NewListHostsParams()
+	ph.SetZoneid(zoneId)
+	ph.SetHypervisor("KVM")
+	ph.SetResourcestate("Enabled")
+	ph.SetState("Up")
+	listHostsResponse, err := client.Host.ListHosts(ph)
+	Expect(err).To(BeNil(), "error listing hosts")
```

Comment:
do the contents of `err` get logged here? That'd be useful for troubleshooting.

Reply:
Yes:
`  error listing zones
  Expected
      <*errors.errorString | 0xc00154abe0>: {
          s: "CloudStack API error 401 (CSExceptionErrorCode: 0): unable to verify user credentials and/or request signature",
      }
  to be nil`